src/org/zaproxy/zap/utils/EnableableInterface.java

```diff
@@ -0,0 +1,13 @@
+package org.zaproxy.zap.utils;
+
+public interface EnableableInterface {
+
+	boolean isEnabled();
+
+	void setEnabled(boolean enabled);
+
+	int hashCode();

```

Comment:
This method (and equals) could be removed from the interface, not really needed.